x/zoneconcierge/keeper/proof_tx_in_block_test.go

```diff
@@ -54,10 +54,14 @@ func TestProveTxInBlock(t *testing.T) {
 	coins := sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, sdk.NewInt(10)))
 	msg := banktypes.NewMsgSend(val.Address, testNetwork.Validators[1].Address, coins)
 
+	txf, err := tx.NewFactoryCLI(val.ClientCtx, fs)
+
+	require.NoError(t, err)
 	// construct a tx that includes this msg
-	txf := tx.NewFactoryCLI(val.ClientCtx, fs).
+	txf = txf.
 		WithTxConfig(val.ClientCtx.TxConfig).
 		WithAccountRetriever(val.ClientCtx.AccountRetriever)
```

Comment:
Do we need to assign here again, or can simply

```suggestion
	txf.WithTxConfig(val.ClientCtx.TxConfig)
	     .WithAccountRetriever(val.ClientCtx.AccountRetriever)
```

?

Reply:
not sure if I understand correctly, but it seems we need this assignemt as `txf` variable is used in subsequent operations like `txf.Prepare(val.ClientCtx)`, `txf.BuildUnsignedTx(msg)` etc.